retreats/2025.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: event
+title: Suspension Retreat 2025
+image: venues/studio_friction.jpg
```

Comment:
this is a site-level attribute.

this is basically a page's "featured image" --- aka, the image used for link previews on social media, etc.

You can see how this works here:

https://github.com/DenverSuspensionCollective/DenverSuspensionCollective.github.io/blob/main/_includes/head.html#L17